File: Week-1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int change;
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 0);
    int coins = 0;
    while (change != 0)
    {
        if (change >= 25)
        {
            change -= 25;
            coins++;
        }
        else if (change >= 10)
        {
            change -= 10;
            coins++;
        }
        else if (change >= 5)
        {
            change -= 5;
            coins++;
        }
        else
        {
            change--;
            coins++;
        }
    }
    printf("%d\n", coins);
    return 0;
}

File: Week-6/sentimental-cash/sentimental-cash/cash.py
from cs50 import get_float


def main():
    change = -1
    while change < 0:
        change = get_float("Change: ")

    change = change * 100

    coins = 0
    while change > 0:
        if change >= 25:
            change -= 25
        elif change >= 10:
            change -= 10
        elif change >= 5:
            change -= 5
        else:
            change -= 1
        coins += 1

    print(coins)


main()
